.ci/apm-beats-update.groovy

```diff
@@ -35,7 +35,13 @@ pipeline {
       when {
         beforeAgent true
         expression {
-          return isCommentTrigger() || isUserTrigger()
+          def ret = isCommentTrigger() || isUserTrigger()
+          if(!ret){
+            currentBuild.result = 'NOT_BUILT'
+            currentBuild.description = "Build Skipped"
```

Comment:
nit: `Skipped build` or `The build has been skipped`

Reply:
```suggestion
            currentBuild.description = "The build has been skipped"
```